src/planner/SnapConfigurationToConfigurationPlanner.cpp

```diff
@@ -0,0 +1,76 @@
+#include "aikido/planner/SnapConfigurationToConfigurationPlanner.hpp"
+
+#include "aikido/common/VanDerCorput.hpp"
+#include "aikido/constraint/Testable.hpp"
+#include "aikido/statespace/StateSpace.hpp"
+
+namespace aikido {
+namespace planner {
+
+//==============================================================================
+SnapConfigurationToConfigurationPlanner::
+    SnapConfigurationToConfigurationPlanner(
+        statespace::ConstStateSpacePtr stateSpace,
+        statespace::ConstInterpolatorPtr interpolator)
+  : ConfigurationToConfigurationPlanner(std::move(stateSpace))
+  , mInterpolator(std::move(interpolator))
+{
+  if (!mInterpolator)
+  {
+    mInterpolator
+        = std::make_shared<statespace::GeodesicInterpolator>(mStateSpace);
+    // TODO(JS): Not sure if GeodesicInterpolator is a good choice as the
+    // default.
```

Comment:
Can you make an issue stating why? It can get lost here and seems important if there's valid reasons. Thanks!

Reply:
I put this comment without looking at other interpolators. Now I think it's a good choice. Let me delete this comments.